flake.nix

```diff
@@ -4,6 +4,7 @@
   inputs = {
     flake-utils.url = "github:numtide/flake-utils";
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-25.05";
+    nixpkgs-unstable.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
```

Comment:
I've added nixpkgs-unstable as a separate input here, to minimise differences to the previous configuration. However, I would recommend using nixpkgs-unstable for everything, optionally pinning a commit, so that there is only one copy of nixpkgs involved.

Let me know if you'd like me to change this.